src/main/java/frc/robot/subsystems/tank/TankSubsystem.java

```diff
@@ -185,57 +180,34 @@ public void setTankDriveVoltages(double leftVoltage, double rightVoltage) {
 
     @Override
     public void periodic() {
-        // Update odometry readings
-        Rotation2d gyroAngle = getRobotHeading();
-        DifferentialDriveWheelSpeeds wheelVelocities = getWheelSpeeds();
-        double leftDistance = leftEncoder.getPosition();
-        double rightDistance = rightEncoder.getPosition();
-
-        poseEstimator.update(gyroAngle, wheelVelocities, leftDistance, rightDistance);
-
         // Update Shuffleboard entries
-        Pose2d pose = poseEstimator.getEstimatedPosition();
+        Pose2d pose = getRobotPosition();
+
         shuffleboardXEntry.setDouble(pose.getX());
         shuffleboardYEntry.setDouble(pose.getY());
         shuffleboardField.setRobotPose(pose);
     }
 
     /**
      * Gets the estimated current position of the robot.
+     * 
```

Comment:
VSCode autoformat 😕

Reply:
The extra newlines in javadoc and comment wrapping actually makes it harder to read than properly-formatted human written code imo

Reply:
Who is writing the formatting style VSCode is using? I think you can specify your own. Also lint.